testbench.yaml

```diff
@@ -0,0 +1,2 @@
+providers:
+  - Laravel\Dusk\DuskServiceProvider
```

Comment:
This is not required to fix the bug but would allow us to run and test `dusk:chrome-driver` command while developing the package. Usage:

```bash
vendor/bin/testbench dusk:chrome-driver --all
```